app/models/asset.rb

```diff
@@ -16,7 +16,7 @@ class Asset < ActiveRecord::Base
   has_many :comments, as: :commentable, dependent: :destroy
 
   has_many :listeners,
-    -> { distinct.order('listens.created_at DESC').limit(20) },
+    -> { select("distinct listens.*").order('listens.created_at DESC').limit(20) },
```

Comment:
FYI, this broke production. Not sure why it was added, or if it was intentional... but it resulted in getting back fake users like this:

```
=> #<ActiveRecord::Associations::CollectionProxy [#<User id: 7599463, created_at: "2018-09-29 22:16:11", updated_at: "2018-09-29 22:16:11", country: nil, city: nil>, #<User id: 7599462, created_at: "2018-09-29 21:15:19", updated_at: "2018-09-29 21:15:19", country: nil, city: nil>]>
```

Reply:
Weird... `distinct.order('')` was breaking locally... maybe version discrepancy? but i doubt that 🤔